src/main/java/ui/UI.java

```diff
@@ -268,7 +268,10 @@ private boolean isCloseOnQuit() {
         return commandLineArgs.getOrDefault("closeonquit", "false").equalsIgnoreCase("true");
     }
 
-    public void quit() {
+    public void quit(boolean isLogout) {
```

Comment:
This will result in some code repetition because I need to call `browserComponent.onAppQuit()` before `browserComponent.cleanChromeCustomProfile()`, reason being if I clean the profile before the quit, some files would not be cleaned.